lib/shared/screens/admin/shared/helpers/map-settings-ids.js

```diff
@@ -0,0 +1,3 @@
+import {map, compact} from 'lodash';
+
+export default (options) => compact(map(options, (option) => option.id));
```

Comment:
This implementation is quite weak. Options lists might have deep options on properties `unlocks` and `options`. Also you can use native `Array.prototype.map`